### PR TITLE
Added snap.manifest.json specification

### PR DESCRIPTION
**Pull request by ritave**

### CODE REVIEW

**Comment by ritave** [comment]
Note this SIP is instantly marked as Final since the specification is already in use

File: SIPS/sip-9.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+---
+
+## Abstract
+
+Specification of `snap.manifest.json` file in version 0.1.
+
+## Motivation
+
+The goal of this SIP is to supersede [Snaps Publishing Specification v0.1](https://github.com/MetaMask/specifications/blob/c226cbaca1deb83d3e85941d06fc7534ff972336/snaps/publishing.md), and move Snaps specifications into one place - Snaps Improvement Proposals.
+It additionally provides a JSON Schema for the manifest.
```
Comment (rekmarks) [inline SIPS/sip-9.md:16]:
```suggestion
The goal of this SIP is to supersede [Snaps Publishing Specification v0.1](https://github.com/MetaMask/specifications/blob/c226cbaca1deb83d3e85941d06fc7534ff972336/snaps/publishing.md), and move Snaps specifications into one place - Snaps Improvement Proposals.
```

File: SIPS/sip-9.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+## Specification
+
+> Such sections are considered non-normative.
```
Comment (rekmarks) [inline SIPS/sip-9.md:20]:
```suggestion
> Indented sections like this are non-normative.
```

**Review by rekmarks — CHANGES_REQUESTED** [review]

File: SIPS/sip-4.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ The snap's `package.json` MUST adhere to `package.json` schema as [defined by th
   - `.description` - The wallet MAY use this field to display information about the snap to the user.
   - `.version` - MUST be a valid [SemVer](https://semver.org/spec/v2.0.0.html) string. DApps MAY request specific version ranges. If a mismatch occurs between requested version range and the version inside the fetched snap, that snap MUST NOT be installed.
     > Additionally, wallet MAY allow updating snaps to the requested version in some schemes.
-    > For example, if the installed snap is `npm:my-snap@1.0.0` while the dapp requests `npm:my-snap@^2.0.0`, the wallet will try to get the newest version from [npm.js](https://npm.js) and will update the snap a version that satisfies `^2.0.0`.
+    > For example, if the installed snap is `npm:my-snap@1.0.0` while the dapp requests `npm:my-snap@^2.0.0`, the wallet will try to get the newest version from [npm.js](https://npmjs.com) and will update the snap a version that satisfies `^2.0.0`.
```
Comment (rekmarks) [inline SIPS/sip-4.md:43]:
```suggestion
    > For example, if the installed snap is `npm:my-snap@1.0.0` while the dapp requests `npm:my-snap@^2.0.0`, the wallet will try to get the newest version from [npm](https://npmjs.com) and will update the snap a version that satisfies `^2.0.0`.
```

**Review by rekmarks — APPROVED** [review]
Nice!

**Pull request merged**